<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_add_form</class>
 <widget class="QWidget" name="student_add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);
color: rgb(255, 255, 255);
font-family: &quot;Century Gothic&quot;;
font-size:15px</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:rgb(36, 31, 49);
color:rgb(255, 255, 255);
font-size:15px
}
            QPushButton {
                background-color: rgb(61, 56, 70);
                color:rgb(255, 255, 255);
                border: none;
                padding: 8px 16px;
                border-radius: 4px;
					font-size:20px
            }
            QPushButton:hover {
                background-color:rgb(192, 191, 188);
					color:black;
            }
QPushButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}
QToolButton:pressed {
    background-color: rgb(153, 193, 241);  /* Change the pressed color as desired */
}
            QToolButton {
                background-color: rgb(53, 132, 228);
                color:black;
                border: none;
                padding: 8px 16px;
                border-radius: 4px;
            }
            QToolButton:hover {
                background-color:rgb(246, 245, 244);
					color:black;
            }
QToolButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}
QPushButton:pressed {
    background-color: rgb(153, 193, 241);  /* Change the pressed color as desired */
}

</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="firstname_input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Phone:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="phone_input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Last name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QPushButton" name="take_photo_button">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color:rgb(61, 56, 70);
	
	color: rgb(255, 255, 255);
	font-size: 20px

}

QPushButton:hover {
    background-color: rgb(119, 118, 123);  /* Change the hover color as desired */
}

QPushButton:pressed {
    background-color: rgb(192, 191, 188);  /* Change the pressed color as desired */
}

QPushButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}</string>
         </property>
         <property name="text">
          <string> Take a photo</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../icons/cam.png</normaloff>../../icons/cam.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>First name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="lastname_input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="Cancel_button">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color:rgb(61, 56, 70);
	
	color: rgb(255, 255, 255);
	font-size: 20px

}

QPushButton:hover {
    background-color: rgb(119, 118, 123);  /* Change the hover color as desired */
}

QPushButton:pressed {
    background-color: rgb(192, 191, 188);  /* Change the pressed color as desired */
}

QPushButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}</string>
         </property>
         <property name="text">
          <string> Cancel</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../icons/cancel.png</normaloff>../../icons/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="add_button">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color:rgb(61, 56, 70);
	
	color: rgb(255, 255, 255);
	font-size: 20px

}

QPushButton:hover {
    background-color: rgb(119, 118, 123);  /* Change the hover color as desired */
}

QPushButton:pressed {
    background-color: rgb(192, 191, 188);  /* Change the pressed color as desired */
}

QPushButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}</string>
         </property>
         <property name="text">
          <string> Add</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../icons/ok.png</normaloff>../../icons/ok.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="email_input">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="take_button">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color:rgb(61, 56, 70);
	
	color: rgb(255, 255, 255);
	font-size: 20px

}

QPushButton:hover {
    background-color: rgb(119, 118, 123);  /* Change the hover color as desired */
}

QPushButton:pressed {
    background-color: rgb(192, 191, 188);  /* Change the pressed color as desired */
}

QPushButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}</string>
         </property>
         <property name="text">
          <string> Capture</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../icons/take.png</normaloff>../../icons/take.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="change_cam_button">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color:rgb(61, 56, 70);
	
	color: rgb(255, 255, 255);
	font-size: 20px

}

QPushButton:hover {
    background-color: rgb(119, 118, 123);  /* Change the hover color as desired */
}

QPushButton:pressed {
    background-color: rgb(192, 191, 188);  /* Change the pressed color as desired */
}

QPushButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}</string>
         </property>
         <property name="text">
          <string>Change camera</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../icons/cam_switch.png</normaloff>../../icons/cam_switch.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="camera_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
